fiddler/datadog_checks/fiddler/check.py

```diff
@@ -0,0 +1,142 @@
+
+import fiddler as fdl
+
+from datadog_checks.base import AgentCheck
+
+# from datadog_checks.base.utils.db import QueryManager
+# from requests.exceptions import ConnectionError, HTTPError, InvalidURL, Timeout
+# from json import JSONDecodeError
+
+# We will get one hours worth data in chunks and process them.
+bin_size = 3600
+
+# metrics_list = ['accuracy', 'histogram_drift', 'output_average', 'feature_average', 'traffic_count']
+metrics_list = ['accuracy']
+tags = list()
+
+# Connection parameters for Fiddler. Currently hard coded to get some testing done. Will be parameterizing this later.
+URL = 'https://demo.trial.fiddler.ai'
+ORG_ID = 'demo'
+AUTH_TOKEN = 'K4ph7ORDcIO2xVIEA6KxL1o1zHjZockgurhCOZOUSVs'
+
+client = fdl.FiddlerApi(url=URL, org_id=ORG_ID, auth_token=AUTH_TOKEN)
+
+
+def create_tags(**tags_in):
+    tags.clear()
+    tags.append("project:" + tags_in['project'])
+    tags.append("model:" + tags_in['model'])
+    if len(tags_in) > 2:
+        tags.append("feature:" + tags_in['feature'])
+
+
+class FiddlerCheck(AgentCheck):
+
+    # This will be the prefix of every metric and service check the integration sends
+    __NAMESPACE__ = 'fiddler'
+
+    def __init__(self, name, init_config, instances):
+        super(FiddlerCheck, self).__init__(name, init_config, instances)
+
+        # Use self.instance to read the check configuration
+        # self.url = self.instance.get("url")
+
+    def check(self, _):
+        # Iterate through the projects and the models and push data into Fiddler
+        try:
+            project_path = ['list_projects', ORG_ID]
+            result_all = client._call(project_path)
+            print("Projects: ", result_all["projects"])
+
+            #            start_time = time.time() * 1000 - (bin_size * 1000)
+            start_time = (time.time() * 1000) - (bin_size * 1000)
+            end_time = time.time() * 1000
+            print("Start time is : ", start_time)
+            print("End time is : ", end_time)
+
+            # Iterate through all of the projects within the Fiddler instance and get the metrics
+            for project in result_all["projects"]:
+                models = project["models"]
+
+                # Iterate through all of the models within a project
+                for model in models:
+                    print("Model: ", model["id"])
+                    for metric in metrics_list:
+                        print("Metric is :", metric)
+                        json_request = {
+                            "metric": metric,
+                            "time_range_start": start_time,
+                            "time_range_end": end_time,
+                            "bin_size": bin_size,
+                            "prediction": '_',
+                        }
+                        agg_metrics_path = ['aggregated_metrics', ORG_ID, project["name"], model["id"]]
+                        print("ProjectModel: ", project["name"], model["id"], metric)
```

Comment:
```suggestion
                        self.log.debug("ProjectModel: ", project["name"], model["id"], metric)
```
Instead of `print()`, let's use `self.log`. See [here](https://github.com/DataDog/integrations-core/blob/080fd6197d51f95152dbb53eca266780a200409d/tls/datadog_checks/tls/tls_remote.py#L56) for example.